pkgs/top-level/all-packages.nix

```diff
@@ -23190,6 +23190,8 @@ in
 
   dcm2niix = callPackage ../applications/science/biology/dcm2niix { };
 
+  deeptools = python3Packages.callPackage ../applications/science/biology/deeptools { };
```

Comment:
In case of applications, pass in `python3` instead of using `python3Packages.callPackage`.